infra/gcp/terraform/k8s-infra-oci-proxy-prod/oci-proxy.tf

```diff
@@ -79,8 +79,9 @@ resource "google_cloud_run_service" "oci-proxy" {
   template {
     metadata {
       annotations = {
-        "autoscaling.knative.dev/maxScale" = "5" // Control costs.
+        "autoscaling.knative.dev/maxScale" = "10" // TODO: adjust to control costs
         "run.googleapis.com/launch-stage"  = "BETA"
+        "run.googleapis.com/cpu-throttling" = false
```

Comment:
I don't think we want to do this one.
CPU throttle down when no requests and put the instance into idle is a cost feature we want and throttle up should be ~instant.

Reply:
removed.